README.md

```diff
@@ -1,4 +1,44 @@
 # State-of-the-art Question Answering With Streamlit and HuggingFace
 
-This is the code accompanying [this post](https://www.mihaileric.com/posts/state-of-the-art-question-answering-streamlit-huggingface/).
+This is the code accompanying [this post](https://www.mihaileric.com/posts/state-of-the-art-question-answering-streamlit-huggingface/). You can try the app [here](https://wikipedia-transformers-qa.herokuapp.com/). Built using Streamlit and deployed on Heroku
```

Comment:
If the plan is to actually keep the Heroku app running, it's fine to keep this here. Otherwise if this is just a self-serve option, I would modify this to say "This setup enables you to deploy a Heroku app for the setup described in the post")

Reply:
I intend to keep it running by utilising the free dyno hours. The dyno sleeps after 30 minutes of inactivity but heroku spins up a dyno as soon as it receives a request (serve on demand). It would allow them to test it out easily (primary reason for this PR) w/o any effort. So, I think it's still better to keep the sentence as is. Besides, I haven't added steps to deploy the Heroku app in README, so it doesn't make much sense to change it to "This setup enables you to deploy a Heroku app for the setup described in the post". 

Let me know your thoughts!

Reply:
Sounds good. As long as the app is live before this gets merged in, so people aren't confused by the wording.